schema.sql

```diff
@@ -7,3 +7,11 @@ CREATE TABLE animals (
     NEUTERED BOOLEAN,
```

Comment:
- [ ] You should add a new table with the name owners with the following columns. 
- [ ] You should add a new table with the name `species` with the following column.